src/components/layout/Navbar/index.tsx

```diff
@@ -77,7 +78,9 @@ const Navbar = ({ user }: NavbarProps) => {
         </Link>
         {/* Desktop Nav Links */}
         <nav className={styles.portalLinks}>
-          <Link href={config.homeRoute}>Events</Link>
+          <Link href={config.homeRoute}>Dashboard</Link>
+          <p aria-hidden>·</p>
+          <Link href={config.eventsRoute}>Events</Link>
           <p aria-hidden>·</p>
           <Link href="/leaderboard">Leaderboard</Link>
           <span aria-hidden>·</span>
```

Comment:
this isn't part of the PR, but it's weird that we use a mix of `p` and `span` for these dot separators